index.html

```diff
@@ -1,14 +1,322 @@
+              distinctio
+              earum
+              suscipit. Lorem ipsum, dolor sit amet consectetur adipisicing elit. Necessitatibus, aliquam recusandae
+              perferendis molestiae numquam eius expedita vitae, laboriosam est facilis autem nobis ad labore quos
+              odio ipsum! Ducimus, error aliquid. Lorem ipsum dolor sit amet consectetur adipisicing elit. Maiores
+              eveniet earum iusto beatae quia labore aliquid saepe ducimus eligendi, debitis quisquam nisi odio aut
+              quibusdam dolorum. Dolorum amet itaque vel? Lorem ipsum dolor, sit amet consectetur adipisicing elit.
+              Ut
+              optio vero sequi reprehenderit quam sint facilis accusantium, praesentium voluptas explicabo. Adipisci
+              quibusdam blanditiis eligendi voluptatum repudiandae, tenetur repellat numquam eum. Lorem ipsum dolor
+              sit amet consectetur adipisicing elit. Obcaecati dolores reiciendis amet quisquam nulla necessitatibus
+              veritatis ab aperiam cum optio rem, nam vitae quaerat. Nesciunt molestias quia et maiores est. Lorem
+              ipsum, dolor sit amet consectetur adipisicing elit. Aliquid aut maiores impedit. Quidem inventore
+              neque
+              laborum autem quasi sapiente nostrum totam fugiat ut veniam? Eaque delectus iure temporibus nulla
+              veritatis? Lorem ipsum dolor sit amet consectetur adipisicing elit. Necessitatibus, illo quo nulla
+              ullam
+              et aspernatur tempora eaque, voluptatibus harum magnam voluptatum illum aliquam sunt. Veniam maxime
+              vel
+              dignissimos quibusdam voluptates. Lorem ipsum dolor sit amet consectetur, adipisicing elit. Maiores
+              corporis suscipit rerum iure fugiat doloremque illum quidem consectetur, officiis illo, quis est
+              harum!
+              Nesciunt consequuntur, fuga aut dolor delectus neque? Lorem ipsum dolor sit amet consectetur
+              adipisicing
+              elit. Laudantium, illum ipsum repellat commodi quae beatae libero itaque. Quaerat aut a magnam
+              dignissimos dolore iste, at dicta in, aliquid consequatur deserunt!</p>
+            <a href='./continue-reading.html'><button class='tell-me-button'>Continue Reading</button></a>
+          </div>
+        </div>
+      </div>
+
+
+
+
+      <!-- Featured Content SideBar -->
+      <section class="sidebar">
+        <div class="dates">
+          <div class='date-post'>
+          </div>
+          <a href='./blog-post.html'><button class='tell-me-button'>JULY 30</button></a>
+          <h3 class='date-title'>JUST ANOTHER <span id='blue-color'>POST</span></h3>
+          <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptates culpa pariatur eos, tempora
+            nihil
+            maiores, ullam omnis impedit libero eveniet deleniti atque error?</p>
+        </div>

```

Comment:
Your `sidebar` class has a few issues, as I'm sure you know. The date should be left-aligned, not spanning the entire width of the column, which would be made apparent/easier by adjusting the width of the column as a whole.  I think the main issue is how small width is in the sidebar as a whole.  Consider shifting the ratio from 70-30, and removing the padding from the `sidebar` class.  That additional margin is one of the reasons why your columns will overlap with one another when scaling down.